app/build.gradle

```diff
@@ -187,6 +190,13 @@ dependencies {
 
     implementation project(path: ':bandwidth-impl')
 
+    implementation project(path: ':secure-storage-api')
+    implementation project(path: ':secure-storage-impl')
+    implementation project(path: ':secure-storage-store')
```

Comment:
Ditto, the store should not be exposed.

Reply:
As per above comment, we decided this is fine to do